biz.aQute.bnd.exporters/test/biz/aQute/bnd/exporters/subsystem/SubsystemTest.java

```diff
@@ -0,0 +1,170 @@
+package biz.aQute.bnd.exporters.subsystem;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+import org.osgi.service.subsystem.SubsystemConstants;
+
+import aQute.bnd.build.Project;
+import aQute.bnd.build.Run;
+import aQute.bnd.build.Workspace;
+import aQute.bnd.exporter.subsystem.SubsystemExporter;
+import aQute.bnd.osgi.Jar;
+import aQute.bnd.osgi.JarResource;
+import aQute.bnd.osgi.Resource;
+import aQute.lib.io.IO;
+import junit.framework.TestCase;
+
+public class SubsystemTest extends TestCase {
+	private String	proj_bundle_1		= "proj_bundle_1";
+	private String	proj_bundle_2		= "proj_bundle_2";
+
+	private String	proj_bundle_1jar	= proj_bundle_1 + "-0.0.1.jar";
+	private String	proj_bundle_2jar	= proj_bundle_2 + "-0.0.1.jar";
+
+	@Override
+	protected void setUp() throws Exception {
+		File result = IO.getFile(RESULTS_PATH);
+
+		if (result.exists()) {
+			Files.walk(result.toPath())
+				.sorted(Comparator.reverseOrder())
+				.map(Path::toFile)
+				.forEach(File::delete);
+		}
+		Workspace ws = getWS();
```

Comment:
As I mentioned above about the .gitignore file, the test case Before method needs to copy the workspace into an output folder and use the workspace in the output folder to avoid (1) writing to a src folder and (2) avoid file locking issues on Windows.